make/compiler/Makefile.pgi

```diff
@@ -51,6 +51,8 @@ GEN_DYNAMIC_FLAG = -Bdynamic
 LIB_STATIC_FLAG =
 LIB_DYNAMIC_FLAG = -shared
 
+# TODO: set target architecture for optimization
+
```

Comment:
I've added that (I'll push in a second), but --no-ieee-floats --ieee-floats wouldn't result in both of these options getting thrown to the backend compiler, since they just toggle the variable inside the Chapel compiler and then that chooses one or the other to emit.